src/main/java/de/tum/in/www1/artemis/web/rest/ModelingExerciseResource.java

```diff
@@ -261,7 +274,11 @@ public ModelingExerciseResource(ModelingExerciseRepository modelingExerciseRepos
         ObjectNode data = objectMapper.createObjectNode();
         data.set("participation", objectMapper.valueToTree(participation));
         if (participation.getResults().size() > 0) {
-            Result result = participation.getResults().iterator().next();
+            Set<Result> resultSet = participation.getResults();
```

Comment:
we need this functionality more often. Can we extract a method, e.g. "findLatestResult" or "findFirstResult" into Participation.java?